package.json

```diff
@@ -14,7 +14,7 @@
     "> 1%"
   ],
   "scripts": {
-    "preanalyze-bundles": "webpack --config webpack.cdn.config.js --profile --json > stats.json",
+    "preanalyze-bundles": "webpack --config webpack.standalone.config.js --profile --json > stats.json",
```

Comment:
I guess this needs to be updated with the changes at https://github.com/Automattic/happychat-client/pull/191?

Reply:
It will be updated when merging the other branch.